### PR TITLE
[luci/pass] Revise FuseInstanceNormPass V6

### PR DESCRIPTION
**Pull request by seanshpark**
This will revise FuseInstanceNormPass version_6 to have const_as_beta as zero values.

### CODE REVIEW

**Comment by seanshpark** [comment]
- for #14386
- from #14390

**Comment by seanshpark** [comment]
@jinevening , is this what you want ?

File: compiler/luci/pass/src/FuseInstanceNormPass.cpp [hunk 1]
```diff
@@ -741,11 +731,12 @@ template <> bool InstanceNormPattern::match<InstanceNormPattern::PatternVersion:
...
   CHECK_OR_FALSE(mean_of_ifm == mean_of_ifm_should_be);
 
-  // mul_gamma is absent
-  // const_as_gamma assume to be 1.0
+  // create 1.0 gamma and 0.0 beta
   auto graph = add_as_terminal->graph();
   const_as_gamma = make_const_one(graph, 1.0f);
   const_as_gamma->name(add_as_terminal->name() + "/gamma");
+  const_as_beta = make_const_one(graph, 0.0f);
+  const_as_beta->name(add_as_terminal->name() + "/beta");
```
Comment (jinevening) [inline compiler/luci/pass/src/FuseInstanceNormPass.cpp:739]: I think these lines are unnecessary. *1 and +0 are Nop.

And, `apply` functions for version_5 and version_6 have to be updated too. There are no gamma and beta for those patterns, so `reshape_gamma_beta` function should not be called.

File: compiler/luci/pass/src/FuseInstanceNormPass.cpp [hunk 2]
```diff
@@ -721,16 +721,6 @@ template <> bool InstanceNormPattern::match<InstanceNormPattern::PatternVersion:
...
 
-  // If const_as_beta has shape of '1 x chennel x (1 or input last dimension)'
-  uint32_t input_channel = ifm_circle->dim(1).value();
-  uint32_t input_last_dim = ifm_circle->dim(2).value();
-  const_as_beta = dynamic_cast<luci::CircleConst *>(sub->x());
-  CHECK_OR_FALSE(const_as_beta);
-  CHECK_OR_FALSE(const_as_beta->rank() == 3);
-  CHECK_OR_FALSE(
-    const_as_beta->dim(0).value() == 1 && const_as_beta->dim(1).value() == input_channel &&
-    (const_as_beta->dim(2).value() == 1 || const_as_beta->dim(2).value() == input_last_dim));
```
Comment (jinevening) [inline compiler/luci/pass/src/FuseInstanceNormPass.cpp:732]: We may need to check the shape and the value of sub->x() (`const_as_zero`).

File: compiler/luci/pass/src/FuseInstanceNormPass.cpp (hunk 1 above)
Comment (seanshpark) [reply compiler/luci/pass/src/FuseInstanceNormPass.cpp:739]: > And, apply functions for version_5 and version_6 have to be updated too.

I cannot check for ver 5 as of now and do not want to mix for both version in one PR.

File: compiler/luci/pass/src/FuseInstanceNormPass.cpp (hunk 2 above)
Comment (seanshpark) [reply compiler/luci/pass/src/FuseInstanceNormPass.cpp:732]: So we need to check all the values should be 0.0f too ?

**Comment by seanshpark** [comment]
@jinevening , I think it would be better you to update V6 with your thoughts.

**Comment by jinevening** [comment]
Sorry for late reply. I think we should first determine how to support groupnorm in our backend. If it's done, our task may become clear.

**Pull request closed without merging**